test/controllers/ChoiceControllerSpec.scala

```diff
@@ -169,27 +195,34 @@ class ChoiceControllerSpec extends ControllerLayerSpec with MockCache {
         theCacheUpserted mustBe Cache(pid, DisassociateUcrAnswers())
       }
 
-      "shut MUCR" in {
+      "user chooses shut MUCR" in {
 
         val result = controller().submit(postWithChoice(Choice.ShutMUCR))
 
         status(result) mustBe SEE_OTHER
         redirectLocation(result) mustBe Some(routes.ChoiceController.displayPage().url)
         theCacheUpserted mustBe Cache(pid, ShutMucrAnswers())
       }
+
+      "user chooses view submissions" in {
+
+        val result = controller(SuccessfulAuth()).submit(postWithChoice(Choice.ViewSubmissions))
+
+        status(result) mustBe SEE_OTHER
+        redirectLocation(result) mustBe Some(routes.ChoiceController.displayPage().url)
+      }
     }
 
     "return 400 when invalid" in {
 
-      val result = controller().submit(postRequest)
+      val result = controller().submit(FakeRequest("POST", "/").withCSRFToken)
 
       status(result) mustBe BAD_REQUEST
-      contentAsHtml(result) mustBe page(Choice.form().bind(Map[String, String]()))
     }
 
     "return 403 when unauthenticated" in {
 
-      val result = controller(UnsuccessfulAuth).submit(postRequest)
+      val result = controller(UnsuccessfulAuth).submit(FakeRequest("POST", "/").withCSRFToken)
```

Comment:
Please extract this `FakeRequest` to ControllerLayerSpec, so we will be able to use `postRequest` as empty POST and `postRequest(json)` as post with json